logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/AssetProtoContent.kt

```diff
@@ -0,0 +1,99 @@
+package com.wire.kalium.logic.data.message
+
+import com.wire.kalium.logic.data.conversation.LegalHoldStatus
+
+data class AssetProtoContent(
```

Comment:
This is reproducing the whole protobuf definition I think we can simplify it a lot.

1. Ignore the uploaded status and only map assets which contain remote data

3. Ignore the preview

4. Ignore legal hold (for now)

5. Ignore expectsReadConfirmation (for now)

Maybe something like this:

```kotlin
data class Asset(
   val size: Int,
   val name: String? = null,
   val metadata: Metadata,
   val remoteData: RemoteData
```

Reply:
Also I don't think we should use `Proto` in our mapped class names

Reply:
I thought about adding the whole protobuf definition and extend it to the SQL model so that we have already all the elements in place. Like that, if we want to expand our list of message types we don't have to add these fields. They are initialised to null in any case, so that shouldn't be a problem right?

Reply:
The protobuf definition is over complicated we don't need all the fields. IMO we should add the bare minimum and then expand it as necessary.